src/nls/it/strings.js

```diff
@@ -743,8 +768,18 @@ define({
     "DESCRIPTION_USE_THEME_SCROLLBARS"               : "true per consentire le barre di scorrimento su misura",
     "DESCRIPTION_LINTING_COLLAPSED"                  : "true per non mostrare il pannello del controllo del codice",
     "DESCRIPTION_FONT_FAMILY"                        : "Modifica il font usato in {APP_NAME}",
-    "DESCRIPTION_FONT_SIZE"                          : "Modifica le dimensioni dei caratteri; ad esempio 13px"
+    "DESCRIPTION_FONT_SIZE"                          : "Modifica le dimensioni dei caratteri; ad esempio 13px",
+    "DESCRIPTION_FIND_IN_FILES_NODE"                 : "true per attivare la ricerca basata su Node",
+    "DESCRIPTION_FIND_IN_FILES_INSTANT"              : "true per attivare la ricerca immediata",
+    "DESCRIPTION_FONT_SMOOTHING"                     : "Solo per Mac: \"subpixel-antialias\" per consentire sub-pixel antialiasing o \"antialias\" per la scala dei grigi antialiasing",
+    "DESCRIPTION_OPEN_PREFS_IN_SPLIT_VIEW"           : "falso per disabilitare l'apertura delle preferenze in Split View",
+    "DESCRIPTION_OPEN_USER_PREFS_IN_SECOND_PANE"     : "falso per aprire le preferenze nel pannello sinistra / alto",
+    "DESCRIPTION_MERGE_PANES_WHEN_LAST_FILE_CLOSED"  : "true al collasso riquadri dopo l'ultimo file dal riquadro è chiusa tramite pannello delle intestazioni pulsante di chiusura",
+    "DESCRIPTION_SHOW_PANE_HEADER_BUTTONS"           : "Alternare quando mostrare i pulsanti di visualizzazione flip-view sull'intestazione.",
+    "DEFAULT_PREFERENCES_JSON_HEADER_COMMENT"        : "/*\n * Si tratta di un file di sola lettura con le preferenze supportati\n * by {APP_NAME}.\n * Usa questo file come riferimento per le modifiche\n * file \"brackets.json\" apri ijn un'altro pannello.\n * Per ulteriori informazioni su come utilizzare le preferenze all'interno\n * {APP_NAME}, riferimento al sito web: https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#preferences\n */",
```

Comment:
```
/*
 * Questo è un file non modificabile che contiene le preferenze supportate da
 * {APP_NAME}. Basati su questo file per modificare le preferenze del file
 * "brackets.json" aperto nell'altra scheda.
 * Per ulteriori informazioni sull'utilizzo delle preferenze vedere
 * https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#preferences
 */
```

`/*\n * Questo è un file non modificabile che contiene le preferenze supportate da\n * {APP_NAME}. Basati su questo file per modificare le preferenze del file\n * "brackets.json" aperto nell'altra scheda.\n * Per ulteriori informazioni sull'utilizzo delle preferenze vedere\n * https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#preferences\n */`